<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>wdgProductsComparation</class>
 <widget class="QWidget" name="wdgProductsComparation">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1422</width>
    <height>520</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_7">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="lbl">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Products comparation</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Select a pair of products</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="cmbPairs"/>
       </item>
       <item>
        <widget class="QToolButton" name="cmdPairsNew">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../images/xulpymoney.qrc">
           <normaloff>:/xulpymoney/add.png</normaloff>:/xulpymoney/add.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="cmdPairsDelete">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../images/xulpymoney.qrc">
           <normaloff>:/xulpymoney/list-remove.png</normaloff>:/xulpymoney/list-remove.png</iconset>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <widget class="wdgProductSelector" name="selector1" native="true"/>
           </item>
           <item>
            <widget class="wdgProductSelector" name="selector2" native="true"/>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QToolButton" name="cmdSwap">
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="../images/xulpymoney.qrc">
             <normaloff>:/xulpymoney/swap.png</normaloff>:/xulpymoney/swap.png</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="Line" name="line_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Select a method to compare</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="cmbCompareTypes">
           <property name="currentIndex">
            <number>0</number>
           </property>
           <property name="sizeAdjustPolicy">
            <enum>QComboBox::AdjustToContents</enum>
           </property>
           <item>
            <property name="text">
             <string>Not changed data</string>
            </property>
            <property name="icon">
             <iconset resource="../images/xulpymoney.qrc">
              <normaloff>:/xulpymoney/eye_red.png</normaloff>:/xulpymoney/eye_red.png</iconset>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Scattering product prices</string>
            </property>
            <property name="icon">
             <iconset resource="../images/xulpymoney.qrc">
              <normaloff>:/xulpymoney/eye.png</normaloff>:/xulpymoney/eye.png</iconset>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Scattering product daily gains percentage</string>
            </property>
            <property name="icon">
             <iconset resource="../images/xulpymoney.qrc">
              <normaloff>:/xulpymoney/eye_red.png</normaloff>:/xulpymoney/eye_red.png</iconset>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Controling percentage evolution</string>
            </property>
            <property name="icon">
             <iconset resource="../images/xulpymoney.qrc">
              <normaloff>:/xulpymoney/eye.png</normaloff>:/xulpymoney/eye.png</iconset>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Controlling percentage evolution. Leveraged reduced</string>
            </property>
            <property name="icon">
             <iconset resource="../images/xulpymoney.qrc">
              <normaloff>:/xulpymoney/eye_red.png</normaloff>:/xulpymoney/eye_red.png</iconset>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Controling inverse percentage evolution</string>
            </property>
            <property name="icon">
             <iconset resource="../images/xulpymoney.qrc">
              <normaloff>:/xulpymoney/eye.png</normaloff>:/xulpymoney/eye.png</iconset>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Controling inverse percentage evolultion. Leveraged reduced</string>
            </property>
            <property name="icon">
             <iconset resource="../images/xulpymoney.qrc">
              <normaloff>:/xulpymoney/eye_red.png</normaloff>:/xulpymoney/eye_red.png</iconset>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Spread of prices joining first value with multiplier</string>
            </property>
            <property name="icon">
             <iconset resource="../images/xulpymoney.qrc">
              <normaloff>:/xulpymoney/eye.png</normaloff>:/xulpymoney/eye.png</iconset>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Price Ratio</string>
            </property>
            <property name="icon">
             <iconset resource="../images/xulpymoney.qrc">
              <normaloff>:/xulpymoney/eye_red.png</normaloff>:/xulpymoney/eye_red.png</iconset>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="Line" name="line">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QLabel" name="label_7">
             <property name="text">
              <string>Use data from</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="cmbDateSelector">
             <property name="currentIndex">
              <number>-1</number>
             </property>
             <property name="sizeAdjustPolicy">
              <enum>QComboBox::AdjustToContents</enum>
             </property>
             <item>
              <property name="text">
               <string>Last 7 days</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Last 30 days</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>last 90 days</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Last 365 days</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Last 3 years</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Last 10 years</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>All data</string>
              </property>
             </item>
            </widget>
           </item>
           <item>
            <widget class="QDateEdit" name="deCompare">
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
             <property name="displayFormat">
              <string>yyyy-MM-dd</string>
             </property>
             <property name="calendarPopup">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QLabel" name="label_8">
             <property name="text">
              <string>Group data by</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="cmbGroupBy">
             <property name="currentIndex">
              <number>0</number>
             </property>
             <property name="sizeAdjustPolicy">
              <enum>QComboBox::AdjustToContents</enum>
             </property>
             <item>
              <property name="text">
               <string>Last 7 days</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Last 30 days</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>last 90 days</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Last 365 days</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Last 3 years</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <widget class="Line" name="line_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="cmdReport">
         <property name="toolTip">
          <string>Generate report</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../images/xulpymoney.qrc">
           <normaloff>:/xulpymoney/history2.png</normaloff>:/xulpymoney/history2.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="Line" name="line_4">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="cmdComparationData">
         <property name="toolTip">
          <string>Shows comparation data</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../images/xulpymoney.qrc">
           <normaloff>:/xulpymoney/database.png</normaloff>:/xulpymoney/database.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="cmdComparation">
         <property name="text">
          <string>Compare</string>
         </property>
         <property name="icon">
          <iconset resource="../images/xulpymoney.qrc">
           <normaloff>:/xulpymoney/compare.png</normaloff>:/xulpymoney/compare.png</iconset>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="VCScatter" name="viewScatter" native="true"/>
     </item>
     <item>
      <widget class="VCTemporalSeries" name="viewCompare" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item>
      <widget class="VCTemporalSeriesWithTwoYAxis" name="viewTwoAxis" native="true"/>
     </item>
     <item>
      <widget class="QLabel" name="lblCorrelation">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>wdgProductSelector</class>
   <extends>QWidget</extends>
   <header location="global">wdgProductSelector</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>VCTemporalSeries</class>
   <extends>QWidget</extends>
   <header location="global">myqcharts</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>VCScatter</class>
   <extends>QWidget</extends>
   <header location="global">myqcharts</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>VCTemporalSeriesWithTwoYAxis</class>
   <extends>QWidget</extends>
   <header location="global">myqcharts</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../images/xulpymoney.qrc"/>
 </resources>
 <connections/>
</ui>
